package.json

```diff
@@ -23,7 +23,7 @@
     "rollback": "knex migrate:rollback",
     "rollback-one": "knex migrate:down",
     "start": "node dist/index.js",
-    "test": "set TEST_ENVIRONMENT=true && ts-mocha --require @babel/register --exit test/** test/**/*",
+    "test": "TEST_ENVIRONMENT=true ts-mocha --require @babel/register --exit test/** test/**/*",
```

Comment:
Why ?

Reply:
"set" like this was for me, because I'm on Windows. On another commit i forgot to change it back . So i was thinking that i have to put it back like it was before.

Maybe i'm wrong doing this ?

Reply:
Sandrine told you that it was working on linux so we can let it like that
and if it was a rollback try to use another commit to keep thing as separate as possible